README.md

```diff
@@ -141,44 +146,64 @@ sct_run_batch -config config_sct_run_batch.yml
+
+**TODO: put video here**
+
+Run the following line and specify the .yml list for cord segmentation with the flag `-config`:
+~~~
+uk_manual_correction -config <.yml file> -path-in ~/ukbiobank_results/data_processed -path-out <PATH_DATA>
+~~~
+After all correction are done, you can generate a QC report by adding the flag `-qc-only-` to the command above. Note that SCT is required for generating QC report.
+
+#### 3. Vertebral labeling
+Note that manual labeling uses SCT and the QC report is generated automatically.
+
+**TODO: put video here**
+
+Run the following line and specify the .yml list for vertebral labeling with the flag `-config`:
 ~~~
 uk_manual_correction -config <.yml file> -path-in ~/ukbiobank_results/data_processed -path-out <PATH_DATA>
 ~~~
-The script loop through all the files listed in .yml file and opens an interactive window to either correct manually segmentation or vertebral labeling. Each mannually corrected label is saved under `derivative/labels/`folder at the root of `PATH_DATA` according to the BIDS convention. Each manually-corrected file has the suffix `-manual`.
 
 C2-C3 disc label will be located at the posterior tip of the disc as shown in the following image. 
 
 ![alt text](https://user-images.githubusercontent.com/2482071/100895704-dabf4a00-348b-11eb-8b1c-67d5024bfeda.png)
 
 #### Upload the manually-corrected files
-A QC report of the manually correct files is created in a zip file. To update the dataset, follow this procedure:
-* Commit and push manually-corrected files (placed in folders under `derivatives/labels/`)
-* Create a pull request and add qc zip file in the body of the PR. 
-* If PR is accepted, a new release of the dataset will be created and the qc zip file will be uploaded as a release object.
+
+A QC report of the manually corrected files is created in a zip file. To update the dataset, add all manually-corrected files `derivatives/labels/`,  and include the qc zip file in the body of the PR. See our [internal procedure](https://github.com/neuropoly/data-management/blob/master/internal-server.md#upload) for more details.
+**TODO see if it is possible to inlcude the qc zipe file in PR** 
 
 #### Re-run the analysis
-After all the necessary segmentation and labels are corrected, re-run the analysis (`sct_run_batch`command in **Processing** section). If manually-corrected files exists, they will be used intead of proceeding to automatic segmentation and labeling. Make sure to put the output results in another folder (flag `-path-output`) if you don't want the previous relsults to be overwritten.
+After all the necessary segmentation and labels are corrected, re-run the analysis (`sct_run_batch`command in [Processing](###processing)). If manually-corrected files exists, they will be used intead of proceeding to automatic segmentation and labeling. Make sure to put the output results in another folder (flag `-path-output`) if you don't want the previous relsults to be overwritten. 
```

Comment:
```suggestion
After all the necessary segmentation and labels are corrected, re-run the analysis (`sct_run_batch` command in [Processing](###processing)). If manually-corrected files exist, they will be used instead of proceeding to automatic segmentation and labeling. Make sure to put the output results in another folder (flag `-path-output`) if you don't want the previous results to be overwritten. 
```